rules-tests/Naming/Rector/Assign/RenameVariableToMatchMethodCallReturnTypeRector/Fixture/demo_file.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\Tests\Naming\Rector\Assign\RenameVariableToMatchMethodCallReturnTypeRector\Fixture;
+
+final class DemoFile
+{
+    /**
+     * @var Redis
```

Comment:
It doesn't rename to `$redis` right when i check the test: https://github.com/rectorphp/rector-src/actions/runs/3042029279/jobs/4899754528#step:5:80 i see the variable is still the same `$lala`.

You are saying you are not expecting it to rename but you wrote the test that it is renaming from `$lala` to `$multi`?

Reply:
@kkmuffme the fixture needs to be valid to make it fixable, without `use Redis;` in use statement, it will lookup `Redis` under namespace which not exists, so it goes skipped which is Ok.

You need to:

- add Redis class to stubs dir
- add `use Redis`

then, make a patch :)